src/Uno.UI.Wasm/ts/Windows/Graphics/Display/DisplayInformation.ts

```diff
@@ -0,0 +1,58 @@
+﻿namespace Windows.Graphics.Display {
+
+	export class DisplayInformation {
+		private static readonly DpiCheckInterval = 500;
```

Comment:
I wonder if it's a bit too intense to validate, even if the use has to wait at most a second to get the update DPI, that's probably acceptable.

Reply:
Upped the interval to 1000 ms 👍